HOWTO-activate-jira-github-activity.rst

```diff
@@ -0,0 +1,47 @@
+How To Activate Recording GitHub Activities in JIRA Issues
+==========================================================
+
+Currently we have the code to receive GitHub PR events via webhooks, and
+record them in JIRA. However, we don't want to activate this code yet
+until we've determined that we can tune the workflow to the appropriate
+level of email notifications.
+
+In order to activate this feature:
+
+1. Add the following config vars at Heroku:
+
+   -  [ ] GITHUB\_PERSONAL\_TOKEN
+   -  [ ] GITHUB\_WEBHOOKS\_SECRET
+   -  [ ] JIRA\_ACCESS\_TOKEN
+   -  [ ] JIRA\_ACCESS\_TOKEN\_SECRET
+   -  [ ] JIRA\_OAUTH\_PRIVATE\_KEY
+   -  [ ] JIRA\_SERVER
+
+2. Add ``rqwoker`` dyno type
+
+   ::
+
+       heroku ps:scale rqworker=1
+
+3. Install the webhooks at various repos by using the
+   ``bin/gh_list_repos_with_webhook.py`` command.
+
+4. Add the following webhook configuration to
+   ``openedx_webhooks.webhook_confs``:
```

Comment:
This has to be edited into a .py file, right? I looked in openedx_webooks/webhook_confs.py, and it wasn't clear to me how to combine what's there with this code.